packages/editor-ui/src/composables/useDocumentTitle.ts

```diff
@@ -0,0 +1,25 @@
+import { useSettingsStore } from '@/stores/settings.store';
+
+interface Options {
+	useDefaultPrefix?: boolean;
+}
+
+const DEFAULT_TITLE = 'Workflow Automation';
+
+export function useDocumentTitle({ useDefaultPrefix }: Options = { useDefaultPrefix: true }) {
+	const settingsStore = useSettingsStore();
+	const { releaseChannel } = settingsStore.settings;
+	const defaultPrefix =
+		!releaseChannel || releaseChannel === 'stable' ? 'n8n' : `[${releaseChannel.toUpperCase()}]`;
```

Comment:
AFAIK, even when we add the release channel, `n8n` should still be part of the title.

```suggestion
		!releaseChannel || releaseChannel === 'stable' ? 'n8n' : `[${releaseChannel.toUpperCase()}] n8n`;
```